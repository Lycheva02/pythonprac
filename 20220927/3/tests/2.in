1,32,74,2,56,9,0
90,6,3,2,67,13,2
0,0,2,0,34,15,8
1,2,3,4,5,6,7
92,35,63,84,194,36,9
234,2,3,4,5,87,901
11,111,1111,111,11,1,11111
2,43,92,3,4,5,1
2,3,4,5,9,0,1
1,2,3,4,5,6,7
32,1,3,6,4,5,2
1,2,3,41,2,3,4
1,2,3,4,5,6,7
9,8,9,2,3,4,5
