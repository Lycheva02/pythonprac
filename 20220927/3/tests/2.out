269,419,643,2803,827,681,842
357,4072,8577,3127,642,765,490
123,166,225,1478,177,236,295
211,137,217,292,114,122,129
3316,4803,9816,9323,1929,2092,1809
8804,17470,29942,3103,4133,5349,5399
104918,92051,105157,28375,40402,51759,63727
